uiplib/scrape.py

```diff
@@ -58,19 +59,22 @@ def get_unsplash_image_links(url, no_of_images):
     for a_tag in a_tags:
         image_url = a_tag['href']
         filename = image_url.split('/')[-2] + ".jpg"
-        image_links.append((filename, image_url))
-        if(len(image_links) >= no_of_images):
+        image_links.append({
+                            'name': filename,
+                            'image_url': image_url
+                           })
+        if len(image_links) < no_of_images:
             break
-
     return image_links
 
 
-def get_reddit_image_links(url, no_of_images):
+def get_reddit_image_links(url):
     """Retrieve images from reddit.
 
     Returns a list of tuples containing filename and url of the image.
     """
     # reddit requires .json format for the URL
+    no_of_images = 5
```

Comment:
Why do we need this hard coded value? It should rather be a setting, right?

Reply:
because it doesn't matter to the user anymore. we are doing that just for our convenience. For all the user knows we are downloading 100s of images

Reply:
I'll remove the setting too. But i felt the settings cleanup is its own PR